docs/cse/get-started-with-cloud-siem/introduction-to-cloud-siem.md

```diff
@@ -0,0 +1,263 @@
+---
+id: introduction-to-cloud-siem
+title: Introduction to Cloud SIEM
+sidebar_label: Introduction to Cloud SIEM
```

Comment:
I might truncate the sidebar label so it reads **Introduction** or **Overview** since it looks kind of repetitive in the sidebars. And title can stay as-is.
<img width="306" alt="Screenshot 2023-05-30 at 4 38 07 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/42f0da22-c99b-4af0-b26c-7c5a4a3bc857">

Reply:
I'll leave it as-is on the principle of "every page is page one". If one comes to the article from a search, I want the title to be clear about what the article is for.